tests/smoke/base.sh

```diff
@@ -52,7 +52,9 @@ elif command -v pdm &> /dev/null; then
 elif command -v hatch &> /dev/null; then
   COMMAND_PREFIX="hatch run"
 elif command -v maturin &> /dev/null; then
-  pip install .  
+  pip install -e .
+elif command -v scikit-build-core &> /dev/null; then
```

Comment:
are you sure that this command really exist?

I tried the latest version, and it doesn't have this command. 
so I would recommend something like:

```suggestion
elif [ "$(pip list|grep -c scikit_build_core)" -ne "0" ]; then
```

PS: please ensure always to check if the command exists or not before adding it here.

Reply:
But how it worked for scikit-build?

Reply:
not sure .. but you should always test it locally to be sure that the command works properly